app/templates/components/widgets/forms/time-picker.hbs

```diff
@@ -1,6 +1,4 @@
 {{#if this.icon}}
   <i class="time icon"></i>
 {{/if}}
-<div class="ui input">
```

Comment:
Why?

Reply:
Expected behaviour - Sir we can see that the "ui" amd "input" classes is being applied on the parent div of the input tag. Now we should put both the icon and the input field inside that div element.

![Screenshot_2021-01-01 Input Semantic UI](https://user-images.githubusercontent.com/65965202/103436997-075fb080-4c48-11eb-9aa1-a55e6c711c48.png)

Reality - but now what's happening is there is one div(having classes "ui" and "input") and inside that there is icon and input field (which is also nested in a div having classes "ui" and "input").

<div id="ember649" class="ui input calendar time picker fluid left icon ember-view"> 
     <i class="time icon"></i>
     <div class="ui input">
           <input name="start_time" placeholder="HH:MM" id="ember650" class="ember-text-field ember-view" type="text">
     </div>
</div>

And due to this the div containing input tag is overlapping the icon. and "THAT'S WHY TIME ICON IS HIDDEN FROM THE WEBSITE ITSELF"

The date picker is not having this issue because 

{{#if this.icon}}
  <i class="calendar icon"></i>
{{/if}}
<Input @type="text" @value={{value}} placeholder={{this.placeholder}} @name={{name}} @focus-out={{action "onChange"}} />

we can see that input tag is not inside the "ui input" div , the whole component have the class "ui input" ,
and that's why the date picker looks completely.

and sir that's why i removed the div because it was overlapping the icon which should not happen as we can see that from date picker.

Reply:
@sachinchauhan2889 please look at it , and correct me if i was unable to explain this behaviour.